docs/release/trg-0/trg-1-09.md

```diff
@@ -3,39 +3,59 @@ title: TRG 1.09 - UX consistency / Styleguide for UI
 sidebar_position: 1
 ---
 
-:::caution
-Proposed release date: 15th of January 2024
-:::
+### Introduction
+This document outlines the criteria for a release gateway focused on ensuring that business applications with user interfaces adhere to the established UI/UX styleguide. The gateway serves as a quality checkpoint before the application is released to production or moved to the next stage of deployment.
```

Comment:
rewording proposal:
This TRG addresses business applications with user interfaces. Purpose is for those apps to adhere to the established UI/UX styleguide. All relevant criteria are outlined and must be checked when releasing a new GitHub version. A crosscheck with every PullRequest is optional.